tests/test_impute.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+import pytest
+
+from sklego.impute import SVDImputer
+
+
+@pytest.fixture
+def X_test():
+    np.random.seed(42)
+    return np.random.normal(size=(100, 5))
+
+
+def test_get_kth_approximation_shape(X_test):
```

Comment:
We've got a continous dataset fixture that comes in all shapes and sizes take could be used here.